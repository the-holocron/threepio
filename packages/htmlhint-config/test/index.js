const config = require('../htmlhintrc.json');

describe('Configuration', () => {
	describe('HTMLHint', () => {
		it('should render the configuration', () => expect(config).toMatchSnapshot());
	});
});
