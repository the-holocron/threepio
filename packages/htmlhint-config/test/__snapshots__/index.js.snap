// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Configuration HTMLHint should render the configuration 1`] = `
Object {
  "alt-require": true,
  "attr-lowercase": true,
  "attr-no-duplication": true,
  "attr-no-unnecessary-whitespace": true,
  "attr-unsafe-chars": true,
  "attr-value-double-quotes": true,
  "attr-value-not-empty": true,
  "doctype-first": true,
  "doctype-html5": true,
  "empty-tag-not-self-closed": false,
  "head-script-disabled": false,
  "id-class-ad-disabled": true,
  "id-class-value": "dash",
  "id-unique": true,
  "inline-script-disabled": true,
  "inline-style-disabled": true,
  "space-tab-mixed-disabled": "tab",
  "spec-char-escape": true,
  "src-not-empty": true,
  "style-disabled": true,
  "tag-pair": true,
  "tag-self-close": true,
  "tagname-lowercase": true,
  "title-require": true,
}
`;
